support_for_advertising_use_cases.md

```diff
@@ -165,19 +190,87 @@ These use-cases are more focused on the publisher's perspective.
+| [Revenue management](#revenue-management) | Little support in TURTLEDOVE, as nor reporting nor updating bidding JavaScripts is not real-time. This means delays in both information and action, leading to very complex and inaccurate budget/revenue management. |  |  |
 
 # User Needs
 
-# Current Proposals and Browser Support
+- [Why am I seeing this ad?](#why-am-i-seeing-this-ad)
+- [Opt-out of an advertiser, category or product](#opt-out-of-an-advertiser-category-or-product)
+- [Opt-out of a marketing service](#opt-out-of-a-marketing-service)
+- [Seeing ads relevant to me](#seeing-ads-relevant-to-me)
+- [Smooth user experience](#smooth-user-experience)
 
 | Use-case | Chrome | Safari | Community Proposals |
 |----------|--------|--------|---------------------|
-| [Exclusion Targeting](#exclusion-targeting) | Acknowledgement that this is a valuable use-case and a link to Facebook’s PETREL proposal on this [GitHub Issue](https://github.com/michaelkleber/turtledove/issues/3) | No support | Facebook proposal for “[Private Exclusion Targeting Rendered Exclusively Locally (PETREL)](https://github.com/w3c/web-advertising/blob/master/PETREL.md)” |
-| [Frequency Capping](#frequency-capping) / [Frequency Optimization](#frequency-optimization) | For ads served via TURTLEDOVE interest-group targeting, the proposal offers a way to [handle frequency capping on-device](https://github.com/michaelkleber/turtledove#on-device-auction).  For other types of targeting, while it will not be possible to enforce a hard "frequency-cap" across multiple websites, it might be possible to calibrate a target average frequency model. See discussion about how to do this on the explainer for the [Aggregate Reporting API](https://github.com/csharrison/aggregate-reporting-api#advanced-example-calibrating-a-frequency-capping-model). | No support | |
-| [Why am I seeing this](#why-am-i-seeing-this) | Both the Chrome team and the [Google ads team](https://services.google.com/fh/files/misc/industry_request_for_comment_v1.0.pdf) have on multiple occassions cited a desire show people an explanation of what caused an ad to appear. The [TURTLEDOVE](https://github.com/michaelkleber/turtledove#motivating-use-case) proposal specifically aims to provide people with an accurate answer as to why they are seeing a re-targeting ad. | No support. | |
-| [Hide all ads from business](#hide-all-ads-from-business) | Unclear | No support | |
+|[Why am I seeing this ad?](#why-am-i-seeing-this-ad)| Both the Chrome team and the Google ads team have on multiple occassions cited a desire show people an explanation of what caused an ad to appear. The TURTLEDOVE proposal specifically aims to provide people with an accurate answer as to why they are seeing a re-targeting ad. | No support. |  |
+|[Opt-out of an advertiser, category or product](#opt-out-of-an-advertiser-category-or-product)| Unclear | No support |  |
+|[Opt-out of a marketing service](#opt-out-of-a-marketing-service)| Supported |  |  |
```

Comment:
How is this supported? What supports it? Can you link to a Chrome explainer or issue that shows how this is supported?

Reply:
Chrome engineers mentioned a UI that would allow among other things to opt-out of an entire marketing service.
However, I couldn't find any document giving any details about it. I am going to remove this in the next commit.